feature/aft/aft_base/otg_tests/aft_prefixcounters/README.md

```diff
@@ -4,162 +4,199 @@
 
 IPv4/IPv6 prefix counters
 
-## Procedure
+## Test Setup
+
+### Generate DUT and ATE Configuration
 
 Configure DUT:port1 for IS-IS session with ATE:port1
 *   IS-IS must be level 2 only with wide metric.
 *   IS-IS must be point to point.
 *   Send 1000 ipv4 and 1000 ipv6 IS-IS prefixes from ATE:port1 to DUT:port1.
-*   IS-IS must be up and running, prefixes must be in RIB and FIB.
 
 Establish eBGP sessions between ATE:port1 and DUT:port1.
 *   Configure eBGP over the interface ip.
 *   Advertise 1000 ipv4,ipv6 prefixes from ATE port1 observe received prefixes at DUT.
-*   Validate total number of entries of AFT for IPv4 and IPv6.
-*   Each prefix must have 1 next hop pointing to ATE port1.
 
-Verifications
+## Procedure
+
+*   Gnmi set with REPLACE option to push the configuration DUT.
+*   ATE configuration must be pushed.
+
+## verifications
 
 *   BGP routes advertised from ATE:port1 must have 1 nexthop.
 *   IS-IS routes advertised from ATE:port1 must have one next hop.
+*   Use gnmi Subscribe with ON_CHANGE option to /network-instances/network-instance/afts.
+*   Verify afts prefix entries using the following paths with in a period of 30s.
+
+/network-instances/network-instance/afts/ipv4-unicast/ipv4-entry/state/prefix,
+/network-instances/network-instance/afts/ipv6-unicast/ipv6-entry/state/prefix
+
 
 
 ## AFT-2.1.1: AFT Prefix Counters ipv4 packets forwarded, IS-IS route.
 
-From ATE:port2 send 10000 pps to one of the ipv4 prefix advertise by IS-IS, after 1 minute stop the traffic.
+### Procedure
 
-Verifications
+From ATE:port2 send 10000 packets to one of the ipv4 prefix advertise by IS-IS.
```

Comment:
You could send a very small number of packets, say 1000 or even 100.  This is only a functional test, to verify the feature exists.  Other tests can cover scaling and stress.  Reducing the number of packets will allow the test to run with even less time.

Reply:
10,000 packets are needed to have a sufficient measurable counter value. I dont think so reducing the packet count can reduce the time, IXIA has option to send 1000,10,10000 packets per sec. So there wont be any saving of time. 10,000 is a good number for test.